tracing/http.go

```diff
@@ -0,0 +1,228 @@
+package tracing
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/trace"
+)
+
+const (
+	//RequestHeaderAnnotationPrefix is a prefix which added to each request header attribute
+	RequestHeaderAnnotationPrefix = "request.header."
+
+	//ResponseHeaderAnnotationPrefix is a prefix which added to each response header attribute
+	ResponseHeaderAnnotationPrefix = "response.header."
+
+	//RequestPayloadAnnotationKey is a key under which request payload stored in span
+	RequestPayloadAnnotationKey = "request.payload"
+
+	//ResponsePayloadAnnotationKey is a key under which response payload stored in span
+	ResponsePayloadAnnotationKey = "response.payload"
+
+	//DefaultMaxPayloadSize represent max payload size which will be added to span
+	DefaultMaxPayloadSize = 1024 * 1024
+
+	//ReducedMarkerKey is a key for annotation which will be presented in span in case payload was reduced
+	ReducedMarkerKey = "reduced_payload"
```

Comment:
does it make same to have this key consistent to keys above ("." vs "_"). Maybe use `payload.reduced`.
It is something we invented or key is forced by 3rd party?

Reply:
Thank you, that a good idea